src/Form.js

```diff
@@ -13,6 +13,7 @@ const Form = React.createClass({
         uid: this.props.user.uid,
       },
     });
+    this.refs.text.value = '';
```

Comment:
This is the right solution. Since `this.refs.text.value` now appears twice in this method, did you consider storing it in a variable and referencing it in both places? Does that even work?